tests/server/server-executeQuery-005.phpt

```diff
@@ -0,0 +1,66 @@
+--TEST--
+MongoDB\Driver\Server::executeQuery() takes a read preference
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; NEEDS("REPLICASET"); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(REPLICASET);
+
+$rp = new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY);
+$secondary = $manager->selectServer($rp);
+
+$command = new MongoDB\Driver\Command(array('profile' => 2));
```

Comment:
Does this clear out the profile collection?
If it doesn't, then as long as this test has passed once, it will pretty much always pass again even though it should break, as the record is always there